kuma/wiki/tests/test_templates.py

```diff
@@ -703,7 +703,7 @@ def test_new_revision_POST_document_with_current(self, get_current):
         # cause notification emails to error
         # and stop being sent.
         time.sleep(1)
-        eq_(2, len(mail.outbox))
+        assert 2 == len(mail.outbox)
```

Comment:
The Smallest of Nits: This is what some devs dismissively call Yoda order, because it sounds like Yoda from Start Wars to read it out loud ("assert 2 is the length of the mailbox I do!") rather than the "natural" order (assert the length of the outbox is 2").  There's a technical reason to prefer this order, especially in C (https://stackoverflow.com/q/9331259/10612), but the past Kuma devs have been switching to "natural" order as they touch code, ``assert actual == expected``.

I'm not expecting a change this time, just look for it in future test refactors.